src/main/java/Duke.java

```diff
@@ -1,10 +1,77 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    static final int MAX_COMMANDS = 100;
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        greet();
+        generateOutput();
+    }
+
+    private static Task[] taskList = new Task[MAX_COMMANDS];
+    private static int listCount = 0;
+
+    public static void greet() {
+        String greeting = "____________________________________________________________\n"
```

Comment:
Would it be better if you declared the line -------- as a global variable? I think it would reduce repeated codes!